permissions.md

```diff
@@ -1,11 +1,12 @@
-# Permissions
+﻿# Permissions
 
 ## Version 4.2 permissions changes
 In version 4.2 we have changed the permissions system to be, we believe, better. Here are some changes:
 * There is no longer a default group. We have replaced this with a dynamic system that means that users get default permissions without a group. To list all of the default permissions, please refer to the `_perms list` command!
 * We have introduced negation permissions meaning by putting a `-` before a permission will **deny** users access to the permission. This works in terms of group hierarchy; groups at the top will override groups at the bottom. User permissions will also immediately override group permissions.
 * As mentioned before, groups now follow a hierarchy (Groups at the top being more important). To allow for this order to be changed we have introduced the command `_perms group <group> move <position>` to move the group up and down the chain. This order can bee seen by doing `_perms groups`!
 * We have finally added the ability to do full and accurate permission validation meaning you will never be mistake on a permission again!
+* Added group cloning and renaming. Cloning copes the group and all it's perms (Doesn't copy users) to make making similar groups easier. To clone a group just do `_permissions group <group> clone <new group>`And to rename just do `_permissions group <group> rename <new name>`
```

Comment:
*copies
*to make similar groups easier
*<new_group> (also add spacing after the code end, plus lowercase the a)
*<new_name>